CHANGELOG.md

```diff
@@ -1,3 +1,17 @@
+# dbt_salesforce 0.7.0
+🎉 Salesforce Package Updates 🎉
+
+We are updating the Salesforce package! To improve its utility, the changes include the following:
```

Comment:
Can you reference the PR here and make a blanket statement that this PR involves the following changes.
```suggestion
PR [#30](https://github.com/fivetran/dbt_salesforce/pull/30) includes various updates to the Salesforce package! To improve its utility, the changes include the following:
```